docs/using_eessi/building_on_eessi.md

```diff
@@ -0,0 +1,73 @@
+# Building software on top of EESSI
+
+## Building software on top of EESSI with EasyBuild
+Building on top of EESSI with EasyBuild is relatively straightforward. One crucial feature is that EasyBuild supports building against operating system libraries that are _not_ in a standard prefix (such as `/usr/lib`). This is required when building against EESSI, since all of the software in EESSI is build against the [compatibility layer](../compatibility_layer.md).
+
+### Starting the EESSI software environment
+Start your environment as described [here](../using_eessi/setting_up_environment.md)
+
+### Configure EasyBuild
+To configure EasyBuild, first, check out the [EESSI software-layer repository](https://github.com/EESSI/software-layer.git). We advise you to check out the branch corresponding to the version of EESSI you would like to use.
+
+If you are unsure which version you are using, you can run
+```
+echo ${EESSI_PILOT_VERSION}
+```
+to check it.
+
+To build on top of e.g. version `2023.06` of the EESSI software stack, we check it out, and go into that directory:
+
+```
+git clone https://github.com/EESSI/software-layer/ --branch 2023.06
+cd software-layer
+```
+Then, you have to pick a working directory (that you have write access to) where EasyBuild can do the build, and an install directory (with sufficient storage space), where EasyBuild can install it. In this example, we create a temporary directory in `/tmp/` as our working directory, and use `$HOME/.local/easybuild` as our installpath:
+```
+export WORKDIR=$(mktemp --directory --tmpdir=/tmp  -t eessi-build.XXXXXXXXXX)
+export EASYBUILD_INSTALLPATH="${HOME}/.local/easybuild"
+source configure_easybuild
```

Comment:
The order of lines 27 and 28 need to be swapped. That's because `configure_easybuild` sets `EASYBUILD_INSTALLPATH` to `${EESSI_PREFIX}/software/${EESSI_OS_TYPE}/${EESSI_SOFTWARE_SUBDIR}` without checking if it is already set.

```suggestion
export WORKDIR=$(mktemp --directory --tmpdir=/tmp  -t eessi-build.XXXXXXXXXX)
source configure_easybuild
export EASYBUILD_INSTALLPATH="${HOME}/.local/easybuild"
```